woocommerce/woocommerce-android
# Home screen: Add "Settings" icon on More screen.

⚠️ Please do not merge until https://github.com/woocommerce/woocommerce-android/pull/5638 is merged ⚠️

Closes: #5585 


### Description
This adds the Settings icon on top right of More screen, as well as its functionality.

### Testing instructions


1. Run app, go to the "More" menu (rightmost option in bottom bar). 
2. Ensure Settings icon appear on top right,
3. Tap it, ensure that it opens the Settings screen.
